tokio-util/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# 0.5.0 (October 27, 2020)
+
+### Changed
+
+- **Breaking Change**: upgrade to bytes 0.6. `poll_read_buf()` has been removed
+  in favor of a method on tokio's `AsyncReadExt::poll_read_buf()` method.
```

Comment:
Can you split this into several changes? We also don't need the bold "breaking change".

Reply:
Updated, can you take another look?